src/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL='http://localhost:3001/'
```

Comment:
We don't want to add this in. Usually how .env files work is you have a .env.example file that people can copy into a .env file locally, that way you aren't committing your local setup to GitHub